server/routes/users.ts

```diff
@@ -16,46 +18,94 @@ router.get('/signOut', async (req, res) => {
 
 // POST /signIn
```

Comment:
Couple of things:

in routes/index app.use('*") should load localhost:3000 not a page not found since that's what's normal
shouldn't most of these be under / instead of /users (ie. localhost:3000/signin not localhost:3000/users/sigin)
why is there no GET /signIn
why is there no GET /signUp

Reply:
We can make these /.. instead of /users/.. I just did it this way to break up into other files easier.

Also, signin and signup I thought would be POST since you are sending data for authorization instead of trying to receive data. Is the standard to use GET instead?